connect/resource/usage_file_automation.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+
+"""
+This file is part of the Ingram Micro Cloud Blue Connect SDK.
+Copyright (c) 2019 Ingram Micro. All Rights Reserved.
+"""
+from abc import ABCMeta
+
+from connect.logger import logger
+from connect.models.exception import UsageFileAction, Skip
+from connect.models.usage import FileSchema, File
+from connect.resource import AutomationResource
+
+
+class UsageFileAutomation(AutomationResource):
+    __metaclass__ = ABCMeta
```

Comment:
To declare the class as abstract, since the `process_request` method is not implemented (it has to be implemented in a derived class in the connector).

Reply:
I could have just implemented the method and raise a `NotImplementedError` as it's done in the base `AutomationResource` class, but I wanted to avoid meaningless code duplication, and PyCharm complains if I don't implement the method and don't declare the class as abstract.